docs/specviz/displaying.rst

```diff
@@ -4,9 +4,8 @@
 Displaying Spectra
 ******************
 
-Because of its use of `glue <https://glueviz.org/>`_ as the underlying data-handling layer and its
-applicability in several different contexts, Specviz takes a modular approach
-to displaying data that has been loaded.
+Specviz takes a modular approach to displaying data to support the flexibility of
```

Comment:
```suggestion
Specviz takes a modular approach to displaying data to take advantage of the flexibility of
```
I think this might be closer to what the original was getting at.